src/radical/entk/stage.py

```diff
@@ -166,16 +167,19 @@ def func_post():
 
     @name.setter
     def name(self, value):
-        if isinstance(value, str):
-            if ',' in value:
-                raise ValueError(obj=self._uid,
-                                attribute='name',
-                                actual_value=value,
-                                expected_value="Using ',' in an object's name will corrupt the profiling and internal mapping tables")
-            else:
-                self._name = value
-        else:
-            raise TypeError(expected_type=str, actual_type=type(value))
+        invalid_symbols = punctuation.replace('.','')
+        if not isinstance(value, str):
+            raise ree.TypeError(expected_type=str,
+                                actual_type=type(value))
+
+        if any(symbol in value for symbol in invalid_symbols):
+            raise ree.ValueError(obj=self._uid,
+                                 attribute='name',
+                                 actual_value=value,
+                                 expected_value="Valid object names can " +
+                                 "contains letters, numbers and '.'. Any "
```

Comment:
See the comments made before about the typo, templating and messages

Reply:
Done